extensions/wikia/Chat2/js/views/views.js

```diff
@@ -412,8 +412,6 @@ var NodeChatDiscussion = Backbone.View.extend({
 });
 //TODO: rename it to frame NodeChatFrame ?
 var NodeChatUsers = Backbone.View.extend({
-	actionTemplate: _.template( $('#user-action-template').html() ),
-	actionTemplateNoUrl: _.template( $('#user-action-template-no-url').html() ),
 	initialize: function(options) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "options". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)